packages/zoe/src/contracts/callSpread/percent.js

```diff
@@ -0,0 +1,33 @@
+// some tools to make treating ratios as percents easier
+
+import { assert, details as X } from '@agoric/assert';
+
+import { makeRatio, assertIsRatio } from '../../contractSupport/ratio';
+import { natSafeMath } from '../../contractSupport';
+
+const { subtract } = natSafeMath;
+
+const BASIS_POINTS = 10000n;
+
+// If ratio is between 0 and 1, subtract from 1.
+export function oneMinus(ratio) {
```

Comment:
`oneMinus` is only meaningful if the brands are the same. 
```
1 - 3meters/5seconds --> 2meters/5seconds
```
doesn't make any sense. `oneMinus` should assert that the brands are the same. You should also do it before the `<=` assert since the error message there is only meaningful when the brands are the same.

Reply:
done